src/main/java/tdl/s3/sync/Destination.java

```diff
@@ -41,10 +57,67 @@ public AWSSecretsProvider getSecret() {
         return secret;
     }
 
-    public AmazonS3 buildClient() {
-        return AmazonS3ClientBuilder.standard()
+    public AmazonS3 getClient() {
```

Comment:
`getClient()` is required to mock the S3 client call in the `UnfinishedWritingFileUploadingStrategyTest`.
The `getBucket()` and `getPrefix()` just shorthands the `secrets`.

The `MultipartUploadHelper` class needs the bucket and prefix. For example method `getAlreadyUploadedParts` and `getPartListing`. How do you suggest we hide it inside the `Destination` class?